cmake/TPLs.cmake

```diff
@@ -68,6 +68,9 @@ set(RNGSSE2_ROOT ${TPL_DIR}) # prefer ours
 find_package(RNGSSE2)
 if(RNGSSE2_FOUND AND NOT NO_RNGSSE2)
   set(HAS_RNGSSE2 true)  # will become compiler define in Main/QuinoaConfig.h
+else()
```

Comment:
They are set empty! Otherwise if `RNGSSE2` isn't found, `RNGSSE2_INCLUDES` and `RNGSSE2_INCLUDES` have a value of `NOT-FOUND` and cmake will complain.